frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-actions.feature

```diff
@@ -0,0 +1,123 @@
+@knative-eventing
+Feature: Perform actions on Broker
+              As a user, I should be able to perform actions on broker
+        
+        
+        Background:
+            Given user has installed OpenShift Serverless Operator
+              And user has created Knative Eventing CR
+              And user has created or selected namespace "aut-eventing-broker-actions"
+              And user is at Topology page
+        
+        
+        @regression @to-do @odc-5030
+        Scenario: Context Menu for Broker: KC-01-TC01
+            Given user has created broker "default" in application groupings "app"
+             When user right clicks on "default" broker
+             Then user will see "Edit Application grouping" option
+              And user will see "Add Trigger" option
+              And user will see "Edit labels" option
+              And user will see "Edit annotations" option
+              And user will see "Edit Broker" option
+              And user will see "Delete Broker" option
+
+        
+        @regression @to-do @odc-5030
+        Scenario: Sidebar for the Broker: KC-01-TC02
+             When user clicks on "default" broker to open the sidebar
+             Then user will see "default" as title of broker
+              And user will see the Resources tab
+              And user will see the Details tab
+
+        
+        @regression @to-do @odc-5030
+        Scenario: Details tab for the Broker: KC-01-TC03
+             When user clicks on "default" broker to open the sidebar
+              And user selects Details tab
+             Then user will see name of broker as "default"
+              And user will see namespace of broker as "aut-eventing-broker-actions"
+              And user will see labels and annotations associated with broker
+              And user will see the owner and broker created time
+
+
+        @regression @to-do @odc-5030
+        Scenario: Edit Application grouping to new application action on Broker: KC-01-TC04
+             When user right clicks on "default" broker to open the context menu
+              And user clicks on "Edit Application grouping"
+              And user will click on Application drop down on the modal
+              And user selects "Create Application" from Application drop down
+              And user enters application name as "new-app"
+              And user clicks on Save button
+             Then user will see "new-app" application created
+
+        
+        #Please add the label as test data on line 59 and verify it on line 63
```

Comment:
When someone adds a scenario before this scenario, all the line numbers will make no sense.

Reply:
Okay. I will update the line `Please add the respective test data`